### PR TITLE
OOB: CallApi documentation

### PR DESCRIPTION
**Pull request by guillegr123**

### CODE REVIEW

**Comment by guillegr123** [comment]
I've pushed some small improvements. I think it is now ready to review.

File: docs/appContext/callApi().md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+### `data`
+Here is a list of some common data properties. However, this object may contain other properties besides the listed here, depending on the requested resource.
+
+Key | Description | Type | Default | Required
+:-: | --- | :-: | :-: | :-:
+`accountId` | Kazoo account ID. | `String` | | `false`
+`data` | Data payload to be sent in the request body. | `Object` | | `false`
+`filters` | A map of filters to append to the API URL, where the key is the URL parameter name and the value is the parameter's value. | `Object` | | `false`
+`generateError` | Flag that indicates if the method should display the default error alert, if the request fails. | `Boolean` | `true` | `false`
+`headers` | An object of additional header key/value pairs to send along with the request (e.g. `{ 'Accept': 'application/octet-stream' }`).
```
Comment (azefiel) [inline docs/appContext/callApi().md]:
```suggestion
`headers` | An object of additional header key/value pairs to send along with the request (e.g. `{ 'Accept': 'application/octet-stream' }`). | `Object` | | `false`
```

File: docs/appContext/callApi().md [hunk 2]
```diff
@@ -0,0 +1,168 @@
+title: callApi()
+
+# callApi()
```
Comment (azefiel) [inline docs/appContext/callApi().md]:
```suggestion
# appContext.callApi()
```

File: docs/appContext/callApi().md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+## Syntax
+```javascript
+var self = this;
+
+// ...
+
+self.callApi(params: {
```
Comment (azefiel) [inline docs/appContext/callApi().md]:
```suggestion
appContext.callApi(params: {
```

File: docs/appContext/callApi().md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+Key | Description | Type | Default | Required
+:-: | --- | :-: | :-: | :-:
+`params` | A plain JavaScript object that wraps the arguments for the function. | `Object`([#params](#params)) | | `true`
+
+### `params`
+Key | Description | Type | Default | Required
+:-: | --- | :-: | :-: | :-:
+`resource` | Kazoo SDK method name. | `String` | | `true`
+`apiRoot` | Custom API URL, to use instead of the one configured globally. | `String` | | `false`
+`authToken` | A Kazoo auth token. This will override any previously set auth token, but only for the current request. | `String` | | `false`
```
[the remark this answers is not in the document]
Comment (guillegr123) [reply docs/appContext/callApi().md]: It is not `monster.util.getAuthToken`? I did not found `monster.apps.taxation.getAuthToken()`.

[the remark this answers is not in the document]
Comment (azefiel) [reply docs/appContext/callApi().md]: Yeah you are correct, wrong c/p from my side it seems.

**Pull request merged**